flexget/plugins/parsers/parser_guessit.py

```diff
@@ -220,7 +220,7 @@ def parse_series(self, data, **kwargs):
             guess_result = guessit_api.guessit(native(data), options=guessit_options)
         except GuessitException:
             log.warning('Parsing %s with guessit failed. Most likely a unicode error.', data)
-            guess_result = {}
+            guess_result = MatchesDict()
```

Comment:
Couldn't we just abort here in this case? Something like:
```suggestion
            return SeriesParseResult(data=data, valid=False)
```